### PR TITLE
ci(core): release support pipeline to simplify the release and by creating few artefacts automatically

### PR DESCRIPTION
**Pull request by ideoma**
The pipeline will automatically run when repo is tagged with patter `*.*.*` and will publish docker tag, latest + create / update github release with binaries.

### CODE REVIEW

**Review by bluestreak01 — CHANGES_REQUESTED** [review]
we also need to update release instructions to make use of this pipeline

**Comment by ideoma** [comment]
> we also need to update release instructions to make use of this pipeline

done in [139052e](https://github.com/questdb/questdb/pull/4931/commits/139052e8154d987ca299b1b9f6c37ab90ccac903)

File: artifacts/release/README.md [hunk 1]
```diff
@@ -42,37 +34,23 @@ Note that `-B` flag will make assumptions about release version. Use it if your
...
-git checkout tags/7.1.1
-mvn clean package -DskipTests -P build-web-console,build-binaries
-```
-
 ## Release Docker image
 
-Prune docker images to ensure clean build
+Azure [Build Docker Image](https://dev.azure.com/questdb/questdb/_build?definitionId=22) pipeline will automatically
+build and release the docker image to Docker Hub once the tag is pushed to the repo.
+See previous versions of this readme to find out how to manually build and push the image.
```
Comment (bluestreak01) [inline artifacts/release/README.md:41]: do you mind keeping the manual commands in this version of the readme to save the operator having to use `git` in an event of CI failure

Comment (ideoma) [reply artifacts/release/README.md]: Done in [a7020c7](https://github.com/questdb/questdb/pull/4931/commits/a7020c72bf729ca69510624bf9f1eb5bf9e5f81f)

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**